wordpress-mobile/gutenberg-mobile
# Fix Android crash on device rotation

This PR updates the Gutenberg ref to test https://github.com/WordPress/gutenberg/pull/14576

To test: 
- Check out WPAndroid with the `gutenberg-mobile` submodule checkout on this branch.
- Run the app and instantiate an empty Gutenberg instance.
- Rotate the device/emulator.
- Check that gutenberg doesn't crash.
